package.json

```diff
@@ -9,9 +9,28 @@
     "test": "jest"
   },
   "devDependencies": {
-    "eslint": "^7.21.0",
+    "@babel/core": "^7.16.0",
+    "@babel/eslint-parser": "^7.16.3",
+    "@babel/preset-react": "^7.16.0",
+    "@types/node": "^16.11.12",
+    "@typescript-eslint/eslint-plugin": "^5.6.0",
+    "@typescript-eslint/parser": "^5.6.0",
+    "eslint": "8.2.0",
+    "eslint-config-airbnb": "19.0.4",
     "eslint-config-airbnb-base": "^14.2.1",
-    "eslint-plugin-import": "^2.22.1",
-    "jest": "^26.6.3"
+    "eslint-config-airbnb-typescript": "^16.1.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-html": "^6.2.0",
+    "eslint-plugin-import": "2.25.3",
+    "eslint-plugin-jsx-a11y": "6.5.1",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "7.28.0",
+    "eslint-plugin-react-hooks": "4.3.0",
+    "jest": "^26.6.3",
+    "prettier": "^2.5.1",
+    "typescript": "^4.5.2"
+  },
+  "dependencies": {
+    "yarn": "^1.22.17"
```

Comment:
Yarn must be installed at a machine level, not at a project level, btw, there is just needed one package manager npm or yarn, so there should be just the yarn.lock or the package-lock.json